cds_ils/importer/providers/cds/rules/journal.py

```diff
@@ -85,11 +85,19 @@ def internal_notes(self, key, value):
     return {"value": clean_val("a", value, str, req=True)}
 
 
-@model.over("note", "^935__")
+@model.over("note", "(^935__)|(^866__)")
 @out_strip
 def note(self, key, value):
     """Translates note field."""
-    return clean_val("a", value, str, req=True)
+    arr = [self.get("note", "")]
```

Comment:
naming police 🚓 
maybe something more descriptive for the list name, like
`list_of_notes`, ` list_of_notes_to_concatenate`